sale_blanket_order/readme/USAGE.rst

```diff
@@ -0,0 +1,53 @@
+A new menu in the Sales area is created, allowing users to create new blanket orders.
+
+To create a new Sale Blanket Order go to the sale menu in the Sales section:
+
+.. image:: /sale_blanket_order/static/description/BO_menu.png
```

Comment:
Please use suggested syntax
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/readme/CONFIGURE.rst
to correctly display images in both README and index

Reply:
Thanks @eLBati, I applied changes as you suggested but it only works in https://github.com/tafaRU/sale-workflow/blob/12.0-mig-sale_blanket_order/sale_blanket_order/readme/USAGE.rst
Trying to regenerate the README file I get https://raw.githubusercontent.com/tafaRU/sale-workflow/12.0-mig-sale_blanket_order/sale_blanket_order/README.rst that doesn't work.
Any idea why that might be?

Reply:
Will work when merged

Reply:
@hveficent, ah cool! I'm not aware of it. Do you have any reference?